scripts/plot_network.py

```diff
@@ -23,7 +23,7 @@
 from plot_summary import preferred_order, rename_techs
 from pypsa.plot import add_legend_circles, add_legend_lines, add_legend_patches
 
-plt.style.use(["ggplot", "matplotlibrc"])
+# plt.style.use(["ggplot", "matplotlibrc"])
```

Comment:
haha, yes, throws file not found error if run as script not from Snakefile

Reply:
:D forgot that